thavalon-webapp/src/components/Register.tsx

```diff
@@ -1,32 +1,88 @@
-import React, {useEffect, useState} from 'react';
+import React, {useState} from 'react';

```

Comment:
Upon review, I think React, { useEffect } is fine. I think it's just importing everything to React and useState explicitly. However, we can still use object imports for AccountManager and HttpResponse